BUILD.ghc

```diff
@@ -38,10 +39,14 @@ cc_library(
   strip_include_prefix = glob(["lib/ghc-*/unix-*/include"], exclude_directories=0)[0],
 )
 
+c2hs_toolchain(
+  name ="c2hs",
+  c2hs = "@c2hs//:bin",
+)
+
 haskell_toolchain(
     name = "ghc",
-    c2hs = "@c2hs//:bin",
-    version = "8.2.2",
+    version = "8.6.3",
```

Comment:
I was working ghc 8.6.3 on my own project...i can revert this change if you don't want it. Im a little dissapointed with the hardcoding of the ghc version in both the BUILD and WORKSPACE, but it doesn't look like there are great options.

Reply:
There's a bit of weirdness around the compilation of Hazel-internal cabal2bazel binary - see https://github.com/FormationAI/hazel/issues/53 for related discussion

Reply:
So I guess in a similar vein to https://github.com/FormationAI/hazel/pull/65#discussion_r249980599, should I be using a BUILD.ghc defined in my own repo? It wasn't totally clear me to how much of the WORKSPACE/BUILD items from the hazel repo I needed to get started. Without this bindist I couldn't get anything packages not bundles with GHC to compile

Reply:
Yeah correct - I am not too familiar with this part of hazel but AFAIK a BUILD.ghc file will be necessary to set up the toolchain

Thanks for puzzling through all this and surfacing feedback. We really need to stand up a rules_haskell / hazel quick start repository keyed to a recent stackage snapshot.

Reply:
Ok, maybe I can take a shot at a basic repo with hazel. Im still unable to compile any c2hs code properly...something about my setup isn't picking up my mac's SDK root (/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/) although its finding the threaded-rts and unix-includes properly. Im not sure how to solve this so maybe throwing up a sample repo and you all can help me figure out where its going wrong.

The one in the rules_haskell_examples doesn't seem to be kept up to date and doesn't include any packages from stackage I think

Reply:
nvm was the annoying `BAZEL_USE_CPP_ONLY_TOOLCHAIN=1`